client/js/components/content-slider.js

```diff
@@ -66,12 +74,18 @@ const contentSlider = (el, options) => {
 
     // Place slider elements into DOM
     sliderElements.slidesContainer.parentNode.insertBefore(sliderElements.slider, sliderElements.slidesContainer);
-    sliderElements.slider.appendChild(sliderElements.slidesContainer);
+    sliderElements.slider.appendChild(sliderElements.sliderInner);
+    sliderElements.sliderInner.appendChild(sliderElements.slidesContainer);
     sliderElements.slider.parentNode.insertBefore(sliderElements.sliderControls, sliderElements.slider.nextSibling);
     sliderElements.sliderControls.appendChild(sliderElements.prevControl);
     sliderElements.sliderControls.appendChild(sliderElements.nextControl);
-    sliderElements.prevControl.innerHTML = sliderElements.arrow;
-    sliderElements.nextControl.innerHTML = sliderElements.arrow;
+    if (settings.sliderType === 'gallery') {
+      sliderElements.prevControl.innerHTML = sliderElements.arrowThin;
+      sliderElements.nextControl.innerHTML = sliderElements.arrowThin;
+    } else {
+      sliderElements.prevControl.innerHTML = sliderElements.arrowThick;
+      sliderElements.nextControl.innerHTML = sliderElements.arrowThick;
+    }
```

Comment:
Not too fussed, the only thing is that we have multiple DOM manipulations that aren't packaged in one place fastDOM would help with this).

It'd be interesting to see what you think of #740 as that could probably help with this quite a lot.